yapapi/script/capture.py

```diff
@@ -0,0 +1,59 @@
+from dataclasses import dataclass
```

Comment:
Do I understand correctly that 100% of this was moved without changes from `yapapi.ctx`?

Reply:
Yup, no changes here.